src/cluster/redis_slot.h

```diff
@@ -19,6 +19,7 @@
  */
 
 #pragma once
+#include <cstdint>
```

Comment:
```suggestion
```

Reply:
Please see the result in https://godbolt.org/z/5q6zj5W9s . So I think the header is due.

Reply:
Got it. Thanks for your demo.